src/tf_deadness_analysis.h

```diff
@@ -0,0 +1,80 @@
+/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+/*******************************************************************************
+
+This file is a copy of
+Github repository: https://github.com/tensorflow/tensorflow
+Revision: 6619dd5fdcad02f087f5758083e2585bdfef9e78
+File: tensorflow/tensorflow/compiler/jit/deadness_analysis.h
+
+Github Permanent Link
+https://github.com/tensorflow/tensorflow/blob/6619dd5fdcad02f087f5758083e2585bdfef9e78/tensorflow/compiler/jit/deadness_analysis.h
```

Comment:
No need to provide the Permanent link (as it's breaking the code style guide. You have provided all the relevant information in lines 18-20 already. That should be sufficient.